testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -223,31 +207,21 @@ def test_ensemble_superimposition():
 
     @staticmethod
     def test_ensemble_superimposition_to_reference_non_weighted():
-        ensemble0 = mda.Universe(PSF, DCD)
```

Comment:
coveralls reports that coverage has decreased. That is strange for such a minimal change.